sql/src/main/java/org/apache/druid/sql/calcite/aggregation/builtin/EarliestLatestAnySqlAggregator.java

```diff
@@ -202,12 +202,20 @@ public Aggregation toDruidAggregation(
         theAggFactory = aggregatorType.createAggregatorFactory(aggregatorName, fieldName, null, outputType, -1);
         break;
       case 2:
+        int maxStringBytes;
+        try {
+          maxStringBytes = RexLiteral.intValue(rexNodes.get(1));
+        }
+        catch (AssertionError ae) {
+          plannerContext.setPlanningError("The second argument '%s' to '%s' function is not a literal", aggregateCall.getName(), rexNodes.get(1));
```

Comment:
The arguments to this formatted error message seem reversed.

Reply:
We can also safely say that the argument should be an integer (maybe also mention the name of the argument) rather than saying that it should be a "literal".